webdriver/tests/bidi/browsing_context/reload/reload.py

```diff
@@ -10,48 +11,65 @@
 async def test_return_value(bidi_session, inline, new_tab):
     url = inline("<div>foo</div>")
     await bidi_session.browsing_context.navigate(context=new_tab["context"],
-                                                 url=url)
+                                                 url=url,
+                                                 wait="complete")
 
     result = await bidi_session.browsing_context.reload(
-        context=new_tab["context"])
+        context=new_tab["context"],
+        wait="complete"
+    )
     assert result == {}
```

Comment:
The result shouldn't be empty anymore. This is going to fail. For example, `result["url"]` should equal `url`.

Reply:
This patch does not include the latest changes to the spec. I'm sure that I referenced https://bugzilla.mozilla.org/show_bug.cgi?id=1851583 in another issue, were I'm going to get this fixed. 

Also note that the discussion on https://github.com/w3c/webdriver-bidi/issues/527 is not done yet and I want to do the changes in one go. So I'm going to wait for a final agreement.